### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DanielTurner**
* Added eslint config (Configuration for eslint)
* Added .gitignore
* Added package.json (NPM package configuration)
* Added Karma config (Testing harness framework)
* Added development of views resources and stylesheets

### CODE REVIEW

File: src/resources/overview/view.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <li>transfer bacteria from their bodies onto anything they touch.</li>
+    </ul>
+</p>
+
+<p>
+    Guide dogs and other assistance animals are allowed in any area used by customers.  Assistance animals are defined by the <i>Disability Discrimination Act 1992</i> (Cth). 
+</p>
+
+<p>
+    Only dogs are allowed in outdoor dining areas if you as the business choose to allow them. Check with your <a href="https://www.qld.gov.au/about/how-government-works/local-government-directory">local council</a> for the rules that apply in your area.
```
Comment (ThrawnCA) [inline src/resources/overview/view.html:18]: "Only dogs are allowed" should be "Dogs are only allowed"

File: src/resources/questions-animal-and-pets/question-2/no.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="alert alert-info" role="alert">
+  <h2><i class="fa fa-exclamation-triangle"></i>Warning!</h2>
+  <p>Your food may become contaminated.</p>
+  <p>Consider these actions:</p>
+  <ul>
+    <li>
+      Have regular pest control done and keep receipts for each time it is done.
+    </li>
+    <li>Regularly check for signs of pests (e.g.: mouse poo, spider webs).</li>
```
Comment (ThrawnCA) [inline src/resources/questions-animal-and-pets/question-2/no.html:9]: Perhaps "droppings" would be more professional?

File: src/resources/questions-animal-and-pets/question-4/yes.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<div class="alert alert-info" role="alert">
+  <h2><i class="fa fa-exclamation-triangle"></i>Warning!</h2>
+  <p>Only dogs are the allowed in outdoor dining areas.</p>
```
Comment (ThrawnCA) [inline src/resources/questions-animal-and-pets/question-4/yes.html:3]: "Only dogs are the allowed" should be "Dogs are only allowed"

File: src/resources/summary/animal-pests/view.html [hunk 4]
```diff
@@ -0,0 +1,189 @@
+<link rel="stylesheet" href="./styles.scss">
+
+<h1>Summary</h1>
+
+<p>
+    Thank you for taking the time to complete the Animals and pests self-assessment.
```
Comment (ThrawnCA) [inline src/resources/summary/animal-pests/view.html:6]: Should "pests" be capitalised?

File: src/resources/summary/animal-pests/view.html [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+    <li>
+        <a href="#">
+            Start another next self-assessment topic
+        </a>
+    </li>
+</ul>
+</p>
+<p>
+    Contact your local council if you would like further 
+    information or resources for your food business
```
Comment (ThrawnCA) [inline src/resources/summary/animal-pests/view.html:39]: Missing full stop.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

**Review by duttonw — APPROVED** [review]

**Comment by duttonw** [comment]
I'm going to merge this so that we get the next lot of changes, please go through @ThrawnCA  comments and implement them if the BA's agree.

**Pull request merged**